src/main/java/org/openpnp/vision/pipeline/CvPipeline.java

```diff
@@ -197,6 +198,14 @@ public void setNozzle(Nozzle nozzle) {
     public Nozzle getNozzle() {
         return nozzle;
     }
+	
+    public void setValue(double value) {
```

Comment:
This does not appear to be used in any stages. It should be removed. If it is required, please commit code that actually uses it and rename it to something more descriptive. "value" is too generic for a property of the pipeline.